src/hackstead/mod.rs

```diff
@@ -1,27 +1,55 @@
-use actix_web::{get, web, HttpResponse};
+use crate::data;
+use actix_web::{get, post, web, HttpResponse};
+use log::*;
 use hcor::{Hackstead, ServiceError, UserContact};
 
+#[cfg(test)]
+mod test;
+
 #[get("/hackstead/")]
 pub async fn get_hackstead(form: web::Json<UserContact>) -> Result<HttpResponse, ServiceError> {
```

Comment:
This should take in a user request, no?

Reply:
I deleted `UserRequest`s a long time ago, felt redundant when we have `UserContact`. Is there some nuance there that I'm not appreciating?

Reply:
IMO `UserRequest` is less data for the client to have to send in, as opposed to the contact data. I see where you're coming from though

Reply:
I mean with `UserRequest` they need to specify either a slack id or an id that we coined
with `UserContact` it's possible to specify both, whereupon we arbitrarily select one or the other (probably the one we have a mongodb index on?)
so I think we ought to just stick with `UserContact` so we never even have to define/expose a `UserRequest`.